cmd/bridge/main.go

```diff
@@ -524,6 +524,8 @@ func main() {
 	httpsrv := &http.Server{
 		Addr:    listenURL.Host,
 		Handler: srv.HTTPHandler(),
+		// Disable HTTP/2, which breaks WebSockets.
```

Comment:
`Disable HTTP/2, which breaks our WebSockets due to a bug in our proxy code.`

We could cross link as well?
https://github.com/openshift/console/blob/master/pkg/proxy/proxy.go#L90

Thinking just a bit more verbosity since it "should just work".

Reply:
I purposely left it a little vague because I'm not certain yet it's a bug in our code. We'll need to dig into it. I think we should track as an issue.

Reply:
Will do: https://issues.redhat.com/browse/CONSOLE-2029